ci/staging/deploy-infrastructure.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+pushd ../../infrasturcture/staging
+    if [ -f terraform.tfstate ]; then
+        echo "  Error, terraform.tfstate file present"
```

Comment:
For consistency, you could rip our error formatting from the other scripts here: https://github.com/newaperio/phoenix_starter/blob/67abc98a572a4b402a1ab2d83ee41969ed2bc8d7/bin/init.sh#L15-L17